portal-impl/src/portal.properties

```diff
@@ -6596,6 +6596,11 @@
     #
     feature.flag.LPD-36719=true
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER3__NUMBER7__NUMBER1__NUMBER0__NUMBER4_
+    #
+    feature.flag.LPD-37104=false
+
```

Comment:
remember to delete in this PR please :smile:

Reply:
Done (https://github.com/liferay-bpm/liferay-portal/pull/4218/commits/617473206c5ef19786f33c4bb70c14569ce83853)